ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -166,3 +166,30 @@ def test_ipa_cert_fix_non_ipa(self):
                                          stdin_text='yes\n',
                                          raiseonerr=False)
         assert result.returncode != 0
+
+
+class TestCertFixKRA(IntegrationTest):
+    @classmethod
+    def uninstall(cls, mh):
+        # Uninstall method is empty as the uninstallation is done in
+        # the fixture
+        pass
+
+    def test_renew_expired_cert_with_kra(self, expire_cert_critical):
+        """Test if ipa-cert-fix renews expired certs with kra installed
+
+        This test check if ipa-cert-fix renews certs with kra
+        certificate installed.
+
+        related: https://pagure.io/freeipa/issue/7885
+        """
+        expire_cert_critical(self.master, setup_kra=True)
+
+        # wait for sometime for certificate expiry
+        time.sleep(10)
+
```

Comment:
Before ipa-cert-fix  can we do ['getcert', 'list'] here to see actually certificates are expired.

Reply:
done